### PR TITLE
Add Grafana Dashboard

### PR DESCRIPTION
**Pull request by 2g4y1**
Add grafana dashbord from here https://github.com/iotaledger/hornet/pull/1674

### CODE REVIEW

File: shimmer-hornet/docker-compose.yml [hunk 1]
```diff
@@ -46,10 +46,11 @@ services:
   ##################################################################
 
   traefik:
-    image: traefik:v2.8.1
+    image: traefik:latest
```
Comment (bogenlos) [inline shimmer-hornet/docker-compose.yml:49]: Could lead to problems if configuration changes. I would prefer a fixed version number.

**Review by bogenlos — CHANGES_REQUESTED** [review]

File: shimmer-hornet/docker-compose.yml (hunk 1 above)
Comment (dlt-green) [reply shimmer-hornet/docker-compose.yml:49]: See it also like bogenlos

Comment (2g4y1) [reply shimmer-hornet/docker-compose.yml:49]: would be v2.8 better? 
Think this should be latest v2.8.x then
https://hub.docker.com/_/traefik

Comment (dlt-green) [reply shimmer-hornet/docker-compose.yml:49]: Same as by other nodes would be good. That we have one standard.

File: shimmer-hornet/docker-compose.yml [hunk 2]
```diff
@@ -110,6 +111,30 @@ services:
...
+      - --event_storage_age_limit=default=0
+      - --store_container_labels=false
+      - --global_housekeeping_interval=30s
+      - --event_storage_event_limit=default=0
+      - --event_storage_age_limit=default=0
+      - --disable_metrics=accelerator,advtcp,cpu_topology,disk,hugetlb,memory_numa,percpu,referenced_memory,resctrl,sched,tcp,udp
+      - --enable_load_reader=true
+      - --docker_only=true          # only show stats for docker containers
+      - --allow_dynamic_housekeeping=true
+      - --storage_duration=1m0s
```
Comment (2g4y1) [inline shimmer-hornet/docker-compose.yml:132]: --publish=8080:8080
When i add this, i get more metrics in Prometheus

File: shimmer-hornet/docker-compose.yml (hunk 1 above)
Comment (2g4y1) [reply shimmer-hornet/docker-compose.yml]: reverted this to 2.8.1 for this pr

**Review by dlt-green — APPROVED** [review]

**Review by dlt-green — APPROVED** [review]
comes from own installer

**Review by dlt-green — APPROVED** [review]

**Pull request merged**